adding rum-cli-examples (#8984)
aws/aws-cli: awscli/examples/rum/untag-resource.rst

**To remove one or more tags from the specified resource.**

> The following ``untag-resource`` removes one or more tags from the specified resource. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``untag-resource`` example removes one or more tags from the specified resource. ::
```

> 	aws rum untag-resource \

Reviewer: Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

		--resource-arn arn:aws:rum:us-west-1:123456789012:appmonitor/MyWebApp \
		--tag-keys Environment

> 

Reviewer: ```suggestion

This command produces no output.

```
